plugins/scaffolder/src/plugin.tsx

```diff
@@ -126,6 +127,7 @@ export const scaffolderPlugin = createPlugin({
     editor: editorRouteRef,
     customFields: customFieldsRouteRef,
     templateForm: templateFormRouteRef,
+    templateExtensions: templateExtensionsRouteRef,
```

Comment:
Should we add the route ref here as well?
https://github.com/backstage/backstage/blob/30471f7661cbbc8fca1d195f4b21c000ef4e45c7/plugins/scaffolder/src/alpha/plugin.tsx#L41

Reply:
If `alpha` means here what it normally does, I guess so 😉 . Will do.